tests/test_accelerator.py

```diff
@@ -472,21 +472,13 @@ def test_accelerator_bnb_cpu_error(self):
         with self.assertRaises(ValueError):
             model = accelerator.prepare(model)
 
-    @require_non_torch_xla
+    @require_cuda
```

Comment:
We should put require_multi_gpu-gpu no ?

Reply:
good catch! I thought require_cuda could replace requre_non_torch_xla. I will keep require_non_torch_xla. Code updated, thx!